### PR TITLE
VTCCA-2292 Property Linking Pages showing duplicate h1 tags

### PR DESCRIPTION
**Pull request by reddy-yattapu**

### CODE REVIEW

File: app/views/main_template.scala.html [hunk 1]
```diff
@@ -71,21 +71,13 @@
 }
 }
 
-@contentHeaderTextAsHtml = @{
```
Comment (DanielConnelly) [inline app/views/main_template.scala.html]: what is this used for reddy before you go just deleting it?

**Comment by habeeb01** [comment]
LGTM 👍

File: app/views/main_template.scala.html (hunk 1 above)
Comment (reddy-yattapu) [reply app/views/main_template.scala.html]: This code is outdated and commit as part of https://jira.tools.tax.service.gov.uk/browse/VTCCA-1232

**Comment by DanielConnelly** [comment]
lgtm 👍

**Pull request merged**